ci-scripts/e2e-cypress.sh

```diff
@@ -77,13 +77,16 @@ echo '-----'
 echo "Building Spartacus storefrontapp"
 npm run build
 
-is_bot_commit() {
+should_run_only_core_e2e() {
     LAST_COMMIT_AUTHOR=$(git log -1 --pretty=format:'%ae')
+    LAST_COMMIT_MESSAGE=$(git log -1 --pretty=format:'%s')
 
     echo "Last commit author: ${LAST_COMMIT_AUTHOR}"
+    echo "Last commit message: ${LAST_COMMIT_MESSAGE}"
 
     if [[ "${LAST_COMMIT_AUTHOR}" == *"dependabot[bot]@users.noreply.github.com" ]] ||
-        [[ "${LAST_COMMIT_AUTHOR}" == *"renovate[bot]@users.noreply.github.com" ]]; then
+        [[ "${LAST_COMMIT_AUTHOR}" == *"renovate[bot]@users.noreply.github.com" ]] ||
+        [[ "${LAST_COMMIT_MESSAGE}" == doc:* ]]; then
```

Comment:
Do we really need such a workaround now?
I believe ideally we should determine the scope of CI jobs to run based on the changed and affected files, but not commit message.

Reply:
btw. I can see a the number of commit messages in develop starting with `doc` among last 1 000 commits is only 32.
and among last 10 000 is only 93.

(source: `git log -n 10000 --oneline | grep -c "^[a-f0-9]\{10\} doc"`)